Android build failed with message "error: An organization slug is required (provide with --org)"

### Environment

How do you use Sentry?
self-hosted (Sentry 22.8.0 f85b134)

Which version of the SDK?

https://github.com/getsentry/sentry-unreal/releases/download/0.1.1/sentry-unreal-0.1.1-engine4.27.zip

How did you install the package? 

Git-URL
https://github.com/getsentry/sentry-unreal/releases/download/0.1.1/sentry-unreal-0.1.1-engine4.27.zip

Which version of Unreal?

4.27.2

Is this happening in Unreal (editor) or on a player like Android, iOS, Windows?

### Steps to Reproduce

1. Install Sentry plugin 
2. Fill required parameter at plugins->sentry
3. I also did Setup Crash Reporter Client https://docs.sentry.io/platforms/unreal/setup-crashreporter/ 
4. Package Android-Shipping Build

### Expected Result

Build should be successful.

### Actual Result

PC build is successful but Android build fails.

### Any logs or screenshots

UATHelper: 패키징 (Android (ETC2)): > Configure project :app
UATHelper: 패키징 (Android (ETC2)): Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/03 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/03 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/03 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
UATHelper: 패키징 (Android (ETC2)): Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
UATHelper: 패키징 (Android (ETC2)): Warning: ����ġ ���� ���(URI: "", ����: "base-extension")�Դϴ�. �ʿ��� ��Ҵ� <{}codename>,<{}layoutlib>,<{}api-level>�Դϴ�.

UATHelper: 패키징 (Android (ETC2)): > Task :app:preBuild
UATHelper: 패키징 (Android (ETC2)): > Task :app:extractProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:preReleaseBuild
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:preBuild UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:preReleaseBuild UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:compileReleaseAidl NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:preBuild UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:preReleaseBuild UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:packageReleaseRenderscript NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:compileReleaseAidl NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseAidl UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:packageReleaseRenderscript NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseRenderscript NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :app:generateReleaseBuildConfig UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:generateReleaseResValues UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:generateReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:processReleaseGoogleServices UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:compileReleaseRenderscript NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:generateReleaseResValues UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:generateReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:packageReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:generateReleaseResValues UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:compileReleaseRenderscript NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:generateReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:packageReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:mergeReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:createReleaseCompatibleScreenManifests UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:extractDeepLinksRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:extractDeepLinksRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:processReleaseManifest UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:extractDeepLinksRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:processReleaseManifest UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:processReleaseManifest UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:compileReleaseLibraryResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:parseReleaseLocalResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:generateReleaseRFile UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:compileReleaseLibraryResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:parseReleaseLocalResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:generateReleaseBuildConfig UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:generateReleaseRFile UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:processReleaseResources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:javaPreCompileRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:compileReleaseJavaWithJavac UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:bundleLibCompileToJarRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:generateReleaseBuildConfig UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:javaPreCompileRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:javaPreCompileRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:compileReleaseJavaWithJavac UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:bundleLibCompileToJarRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseKotlin UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseJavaWithJavac UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseSources UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:mergeReleaseShaders UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:compileReleaseShaders NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :app:generateReleaseAssets UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:generateSentryProguardUuidRelease
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:mergeReleaseShaders UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:compileReleaseShaders NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:generateReleaseAssets UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:packageReleaseAssets UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:mergeReleaseShaders UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:compileReleaseShaders NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:generateReleaseAssets UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:packageReleaseAssets UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:mergeReleaseAssets
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:mergeReleaseGeneratedProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:exportReleaseConsumerProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:mergeReleaseGeneratedProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:exportReleaseConsumerProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:checkReleaseDuplicateClasses UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:mergeReleaseGeneratedProguardFiles UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:processReleaseJavaRes NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:processReleaseJavaRes NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:bundleLibResRelease NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:processReleaseJavaRes NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:bundleLibResRelease NO-SOURCE
UATHelper: 패키징 (Android (ETC2)): > Task :app:mergeReleaseJavaResource UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :downloader_library:bundleLibRuntimeToJarRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :permission_library:bundleLibRuntimeToJarRelease UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:minifyReleaseWithR8 UP-TO-DATE
UATHelper: 패키징 (Android (ETC2)): > Task :app:uploadSentryProguardMappingsRelease FAILED
UATHelper: 패키징 (Android (ETC2)): > compressing mappings
UATHelper: 패키징 (Android (ETC2)): > uploading mappings
UATHelper: 패키징 (Android (ETC2)): error: An organization slug is required (provide with --org)
UATHelper: 패키징 (Android (ETC2)): 62 actionable tasks: 12 executed, 50 up-to-date
UATHelper: 패키징 (Android (ETC2)): Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
UATHelper: 패키징 (Android (ETC2)): Please attach the full debug log to all bug reports.
UATHelper: 패키징 (Android (ETC2)): FAILURE: Build failed with an exception.
UATHelper: 패키징 (Android (ETC2)): * What went wrong:
UATHelper: 패키징 (Android (ETC2)): Execution failed for task ':app:uploadSentryProguardMappingsRelease'.
UATHelper: 패키징 (Android (ETC2)): > Process 'command 'cmd'' finished with non-zero exit value 1
UATHelper: 패키징 (Android (ETC2)): * Try:
UATHelper: 패키징 (Android (ETC2)): Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
UATHelper: 패키징 (Android (ETC2)): * Get more help at https://help.gradle.org
UATHelper: 패키징 (Android (ETC2)): BUILD FAILED in 13s
UATHelper: 패키징 (Android (ETC2)): ERROR: cmd.exe failed with args /c "C:\work\unreal\AvoidNewEngine\Intermediate\Android\armv7\gradle\rungradle.bat" :app:assembleRelease

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,7 @@
 - Fix scoped event/message capturing on Android ([#116](https://github.com/getsentry/sentry-unreal/pull/116))
 - Fix event capturing on Linux ([#123](https://github.com/getsentry/sentry-unreal/pull/123))
 - Fix incomplete type forward declaration ([#125](https://github.com/getsentry/sentry-unreal/pull/125))
+- Fix Android packaging issue ([#133](https://github.com/getsentry/sentry-unreal/pull/133))
 
 ### Dependencies
 

--- a/plugin-dev/Source/Sentry/Sentry_Android_UPL.xml
+++ b/plugin-dev/Source/Sentry/Sentry_Android_UPL.xml
@@ -3,19 +3,19 @@
     <init>
         <log text="Sentry SDK Android UPL initialization"/>
 
-        <setBoolFromProperty result="bUploadSymbols" ini="Engine" section="/Script/Sentry.SentrySettings" property="UploadSymbolsAutomatically" default="true" />
-
-        <setStringFromProperty result="PropertiesFilePath" ini="Engine" section="/Script/Sentry.SentrySettings" property="PropertiesFilePath"/>
-        <setStringReplace result="PropertiesFilePath" source="$S(PropertiesFilePath)" find="(FilePath=&quot;" with=""/>
-        <setStringReplace result="PropertiesFilePath" source="$S(PropertiesFilePath)" find="&quot;)" with=""/>
+        <setBoolFromProperty result="bUploadSymbols" ini="Engine" section="/Script/Sentry.SentrySettings" property="UploadSymbolsAutomatically" default="false" />
     </init>
 
     <prebuildCopies>
         <copyDir src="$S(PluginDir)/Private/Android/Java" dst="$S(BuildDir)/src/io/sentry/unreal" />
-        <copyFile src="$S(ProjectDir)/$S(PropertiesFilePath)" dst="$S(BuildDir)/gradle/sentry.properties" />
         <copyFile src="$S(PluginDir)/../ThirdParty/Android/sentry.jar" dst="$S(BuildDir)/gradle/app/libs/sentry.jar" />
         <copyFile src="$S(PluginDir)/../ThirdParty/Android/sentry-android-core-release.aar" dst="$S(BuildDir)/gradle/app/libs/sentry-android-core-release.aar" />
         <copyFile src="$S(PluginDir)/../ThirdParty/Android/sentry-android-ndk-release.aar" dst="$S(BuildDir)/gradle/app/libs/sentry-android-ndk-release.aar" />
+        <if condition="bUploadSymbols">
+            <true>
+                <copyFile src="$S(ProjectDir)/sentry.properties" dst="$S(BuildDir)/gradle/sentry.properties" />
+            </true>
+        </if>
     </prebuildCopies>
 
     <androidManifestUpdates>

